internal/config/client.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 	"time"
 
+	adminPreview "github.com/mongodb/atlas-sdk-go/admin"
```

Comment:
q: what was the reason behind having the sdk preview with different package name? Was expecting `go.mongodb.org/atlas-sdk/preview/admin`. I guess this simplifies the generation of the preview version from SDK side?

Reply:
in that case we would have to create an official release for preview so it would be in the Go package website.
Using GH module name we don't need to register anything